docs/Advanced Features/workflows.md

```diff
@@ -137,6 +138,23 @@ Package test/package@06b2815 pushed to s3://quilt-sergey-dev-metadata
 Also `default_workflow` can be set in config to specify which workflow will be used
 if `workflow` parameter is not provided.
 
+### Push to bucket
+
+Catalog "[Push to bucket](../Walkthrough/Working with the Catalog.md)" feature can be enabled via workflows config by adding `successors` property:
```

Comment:
Link doesn't work https://github.com/quiltdata/quilt/blob/fcb3172dadfa63be85322e31d567b9d6dbfee05e/docs/Advanced%20Features/workflows.md

Maybe because of quotes